config/environments/development.rb

```diff
@@ -69,4 +69,11 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+  config.after_initialize do
+    Bullet.enable = true
+    # Bullet.sentry = true
+    Bullet.alert = true
```

Comment:
Kindly make false  the alert because it popup the windows with eager loading error